packages/design-system/src/components/FormLabel/FormLabel.test.tsx

```diff
@@ -73,7 +73,11 @@ describe('FormLabel', () => {
   });
 
   it('adds punctuation to requirementLabel when hint is also present', () => {
-    let props = { hint: 'Hint', requirementLabel: 'Optional' };
+    interface PropDef {
```

Comment:
Oh, this is a nice reminder for myself to double check the unit tests when converting to TS. NICE!